cmd/tools/vdoc/resources/doc.js

```diff
@@ -115,3 +175,14 @@ function setupCollapse() {
         });
     }
 }
+
+function debounce(func, timeout) {
+    var timer;
+    return (...args) => {
+        const next = () => func(...args);
+        if (timer) {
+            clearTimeout(timer);
+        }
+        timer = setTimeout(next, timeout > 0 ? timeout : 300);
+    }
+}
```

Comment:
Question: Are we gonna abandon support for ES5-supported browsers in favor of ES6?

Reply:
I wouldnt. Not for such simple things.